src/newmagic.cpp

```diff
@@ -1803,7 +1803,7 @@ void cast_manipulation_spell(struct char_data *ch, int spell, int force, char *a
       send_to_char("You easily dodge it!\r\n", vict);
       act("$n dodges out of the way of a cloud of steam.", TRUE, vict, 0, ch, TO_NOTVICT);
     } else {
-      success -= success_test(GET_BOD(vict) + GET_BODY(vict), force - GET_IMPACT(vict));
+      success -= success_test(GET_BOD(vict) + GET_BODY(vict), force - (GET_IMPACT(vict) / 2));
```

Comment:
What's the source for this?

Reply:
3rd Ed Core, page 196: "Impact Armor protects against damage from Elemental Manipulations, but at only half its normal rating (round down)."

Reply:
Lightning Bolt, Acid Stream, Flamethrower, Laser, Steam, Thunderbolt and Waterbolt should all halve impact armour. Clout doesn't, because it's a Telekinetic Manipulation, not Elemental Manipulation